BitwardenShared/UI/Platform/Settings/Settings/Other/OtherSettingsProcessor.swift

```diff
@@ -72,6 +72,7 @@ final class OtherSettingsProcessor: StateProcessor<OtherSettingsState, OtherSett
             state.clearClipboardValue = services.settingsRepository.clearClipboardValue
             state.isAllowSyncOnRefreshToggleOn = try await services.settingsRepository.getAllowSyncOnRefresh()
             state.isConnectToWatchToggleOn = try await services.settingsRepository.getConnectToWatch()
+            state.shouldShowConnectToWatchToggle = services.systemDevice.isIPhone
```

Comment:
🎨 Add a test for this line in `OtherSettingsProcessorTests -> test_perform_loadInitialValues_success`